examples/playwright/src/theia-quick-command-palette.ts

```diff
@@ -78,4 +78,14 @@ export class TheiaQuickCommandPalette extends TheiaPageObject {
         }
         return command.$('.monaco-list-row.focused .monaco-highlighted-label');
     }
+
+    async visibleItems(): Promise<ElementHandle<SVGElement | HTMLElement>[]> {
```

Comment:
I think, we should add a dedicated test for that new method in `examples/playwright/src/tests/theia-quick-command.test.ts`.

Reply:
Test is added